src/hooks/useStats.ts

```diff
@@ -77,13 +77,14 @@ export function getStats(
 }
 
 export function useAllSpeciesStatsForRegion(
-  region: Region,
-  stats: StatsBySpecies
+  region: Region | null,
+  stats: StatsBySpecies | null
```

Comment:
Ideally, stats cannot be null and the stats prop here comes in as an empty object.